esp-hal-common/src/rsa/mod.rs

```diff
@@ -185,6 +270,23 @@ where
         self.rsa.clear_interrupt();
         Ok(())
     }
+
+    #[cfg(feature = "async")]
+    pub(crate) fn inner(&self) -> &RSA {
```

Comment:
Is there any particular reason we keep adding these methods in our async drivers? They don't seem particularly useful and just kind of add noise.